infra/recipes/terraform/azure/sandbox/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "username" {
+  type        = string
+  description = "The admin username for the virtual machine and MySQL server."
```

Comment:
Seen reference of username is other places as well. If this is common username then should remove mysql from here. Same goes with password as well.

Reply:
Fixed.